src/client/game.cpp

```diff
@@ -3001,8 +2989,13 @@ void Game::updateCameraOffset()
 
 	if (!m_flags.disable_camera_update) {
 		auto *shadow = RenderingEngine::get_shadow_renderer();
-		if (shadow)
+		if (shadow) {
 			shadow->getDirectionalLight().updateCameraOffset(camera);
+			// FIXME: I bet we can be smarter about this and don't need to redraw
+			// the shadow map at all, but this is for someone else to figure out.
+			if (!g_settings->getFlag("performance_tradeoffs"))
```

Comment:
Instead of this, I would prefer to bring back the camera offset that does not change at fixed positions.
Right could move back and forth just one node and keep switching the offset, whereas before you also had to move at least 200 nodes. Then I do not think we need to guard this by "performance_tradeoffs".